jobs/drishya_cards_gold.py

```diff
@@ -0,0 +1,92 @@
+"""
+This script will fetch all the cards of a trello board that according to the below criteria:
+- present with in timeframe
+- Fetch data for Drishya as username
+Save the data to Delta Lake Gold table
+
+"""
+
+import os
+import sys
+from datetime import datetime, timedelta
+import argparse
+from loguru import logger
+from pyspark.sql.functions import col
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from helpers import spark_helper as sh
+from config import delta_table_conf as dc
+
+
+def get_user_activity_cards(spark, start_date, end_date, nameofuser):
+    """
+    Fetch all the cards data of a trello board for the user and specified days and save it into a Delta Lake Gold table
+    :param spark: Spark session
+    :param start_date: trello fetching details start date
+    :param end_date: trello fetching details end date
+    :param nameofuser: the name of user that needs to filter/fetch
+    :return none
+    """
+    path = dc.drishya_cards_gold_path
+    try:
+        # Fetch the cards data from the refined cards silver delta table
+        refined_cards_data = spark.read.format(
+            "delta").load(dc.cards_silver_table_path)
```

Comment:
The source should be cards_unique_table. Can you change the path to cards_unique_silver_table_path.